ui/StatusQ/src/wallet/managetokenscontroller.cpp

```diff
@@ -277,7 +276,11 @@ void ManageTokensController::loadSettings()
     setArrangeByCollection(m_settings.value(QStringLiteral("ArrangeByCollection"), false).toBool());
 
     m_settings.endGroup();
-    setSettingsDirty(false);
+
+    if (result != m_settingsData) {
+        m_settingsData = result;
+        setSettingsDirty(true);
+    }
```

Comment:
Pls make sure that all the changes done in the Manage tokens view is properly reflected in the Custom order of the main wallet view, and vice versa (that Hide token also correctly puts the token in the Hidden section of the settings)

Reply:
@noeliaSD Could you please try to pull latest? I can't reproduce

Reply:
It should be working but I'm refactoring the controller placement in https://github.com/status-im/status-desktop/pull/13319 anyway, so should be safe here (and will change soon anyway)

Reply:
> So the settings seem to be synced but when hidding `collectible` via the main wallet view, it's not immediatly hidden. It seems with the assets works fine. Here a video:
> 
>  Screen.Recording.2024-01-31.at.16.39.20.mov

Fixed! 🥳 It was the filtering expression. I've just moved back to the multiple filters implementation.
Not really a big deal in terms of performance and I wasn't sure it's worth the uglier code to do it all in a single expression.

Thanks!